src/Modules/AuthModule/components/Register/Register.tsx

```diff
@@ -1,7 +1,287 @@
+import React, { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import img1 from "../../../../assets/images/PMS 3.png";
+import Avatar from "../../../../assets/images/Ellipse 1.png";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { AuthContext } from "../../../../context/AuthContext";
 
 export default function Register() {
+ let {baseUrl}= useContext(AuthContext)
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { errors },
+  } = useForm();
+  const Navigate = useNavigate();
+  const [selectedImage, setSelectedImage] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
```

Comment:
I don't like the name `showPassword` as it indicates to the reader that it is a function, not a state variable.
We change it to something like `[isPasswordShown,setIsPasswordShown]`. The new name encapsulates that it is a value and a boolean one.